### PR TITLE
Task001 and Task002 - completed.

### PR DESCRIPTION
**Pull request by kmavropulo**
Task001 and Task002 - solved.

### CODE REVIEW

File: epam-java-cources.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (aabarmin) [inline epam-java-cources.iml:1]: This file should be ignored

File: src/main/java/com/epam/university/java/core/task001/Task001Impl.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
+package com.epam.university.java.core.task001;
+
+public class Task001Impl implements Task001 {
+    String firstNumberMessageForIllegalArgumentException = "first number must be the value";
```
Comment (aabarmin) [inline src/main/java/com/epam/university/java/core/task001/Task001Impl.java:4]: Constants should be declared as public static final members

File: src/main/java/com/epam/university/java/core/task001/Task001Impl.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    String secondNumberMessageForIllegalArgumentException = "second number must be the value";
+    String firstNumberMessageForNumberFormatException = "can't convert first input value to number";
+    String secondNumberMessageForNumberFormatException = "can't convert second input value to number";
+
+    @Override
+    public double addition(String firstNumber, String secondNumber) {
+        if (firstNumber == "") throw new IllegalArgumentException(firstNumberMessageForIllegalArgumentException);
```
Comment (aabarmin) [inline src/main/java/com/epam/university/java/core/task001/Task001Impl.java:11]: Add braces and new line after if ()

**Comment by kmavropulo** [comment]
This branch have been replaced by KMavropulo

**Pull request closed without merging**